cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -15,6 +15,9 @@ class Class extends UserType {
     isClass(underlyingElement(this))
   }
 
+  /** Canonical QL class corresponding to this element. */
```

Comment:
An override of a member predicate inherits its QLDoc from the overridden predicate. You should remove the cut-n-paste QLDoc from overrides to avoid them getting out of sync.

Reply:
Will do, thanks.